graphql.go

```diff
@@ -55,13 +55,16 @@ type Client struct {
 	// To log to standard out, use:
 	//  client.Log = func(s string) { log.Println(s) }
 	Log func(s string)
+	// Client can use this function to supply some logging logic to further debug GraphQL request & response
+	DebugLog func(req *http.Request, resp *http.Response)
```

Comment:
Can you name this something like `OnRetry` since it is invoked on retries and can have other logic than just logging?

Reply:
or `BeforeRetry` might be more appropriate

Reply:
Ok will do.